### ISSUE TITLE
Table2. citation

### ISSUE THREAD

**Issue by nitba** [spurra/vae-hands-3d#12]
Hi @Spurra,
Inreagrd with table 2. in your paper, how you get the number of "8.68" from [38] work, in paper I did not found such a number and when I run his 'eval3d.py' provided script I got different number,

Would you please tell me the setup for his code  you have used to get this number?

Thanks,

**Comment by spurra** [comment]
Hi @Ivamcoder,
I believe I just ran eval3d.py from the provided code of Zimmermann & Brox and got this number. It may be entirely possible that there may be minor discrepancies due to version differences. What number do you get?

**Comment by nitba** [comment]
Hi @spurra ,

This is what I get, so far from `8.56` , 
![image](https://user-images.githubusercontent.com/43126756/63327375-6bdee000-c371-11e9-8576-59355b601ea4.png)

**Comment by nitba** [comment]
One more question is that in `Table2` you have omited `Handness` which represent the left or right hand, 
and the `scale`,

- How you omit the scale , Does it mean that the output hand pose is just palm-relative but not normalized respect to a reference bone? so in this experiment did you use `scale Augmentation for data and keeping the hand bounding box in a range`? if so would you please confirm me that I get correctly.

and

-  if we uniform the size of  hand bounding box mentioned in paper as input into network  how can we say the output of network  has correct scale to be scale invariance , I got confused :(

- when you have used both view of STB dataset , it means that instead of `15K` images for training , you have  `30K` images?

**Comment by spurra** [comment]
> Hi @spurra ,
> 
> This is what I get, so far from `8.56` ,
> ![image](https://user-images.githubusercontent.com/43126756/63327375-6bdee000-c371-11e9-8576-59355b601ea4.png)

Is this using the ground truth hand cropped images?



> One more question is that in `Table2` you have omited `Handness` which represent the left or right hand,
> and the `scale`,
> 
>     * How you omit the scale , Does it mean that the output hand pose is just palm-relative but not normalized respect to a reference bone? so in this experiment did you use `scale Augmentation for data and keeping the hand bounding box in a range`? if so would you please confirm me that I get correctly.
> 
> 
> and
> 
>     * if we uniform the size of  hand bounding box mentioned in paper as input into network  how can we say the output of network  has correct scale to be scale invariance , I got confused :(
> 
>     * when you have used both view of STB dataset , it means that instead of `15K` images for training , you have  `30K` images?

- Yes I left out the scaling for that experiment. I do not divide by the bone length of the reference bone
- Yes, using both views doubles the number of training data. I do not remember if I used both though.

**Comment by nitba** [comment]
Hi @spurra ,

> Is this using the ground truth hand cropped images?

yes,  in `eval3d.py` , I have  `BinaryDbReaderSTB(mode='evaluation', shuffle=False, hand_crop=True, use_wrist_coord=False)`

**Comment by spurra** [comment]
I'm not quite sure where the discrepancy comes from. If I find the time, I'll investigate this.

**Comment by nitba** [comment]
Thank you @spurra
